assignments/objects.js

```diff
@@ -18,28 +18,56 @@ const example = {
 }
 
 // Write your intern objects here:
+let interns = [{"id": 1, "name": "Mitzi", "email": "mmelloy0@psu.edu", "gender": "F"},
+{"id": 2, "name": "Kennan", "email": "kdiben1@tinypic.com", "gender": "M"},
+{"id": 3, "name": "Keven", "email": "kmummery2@wikimedia.org", "gender": "M"},
+{"id": 4, "name": "Gannie", "email": "gmartinson3@illinois.edu", "gender": "M"},
+{"id": 5, "name": "Antonietta", "email": "adaine5@samsung.com", "gender": "F"},]
+
+
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(interns[0].name)
+
 // Kennan's ID
 
+console.log(interns[1].id)
+
 // Keven's email
 
+console.log(interns[2].email)
+
 // Gannie's name
 
+console.log(interns[3].name)
+
 // Antonietta's Gender
 
+console.log(interns[4].gender)
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+interns[1].speak = function sayHello() {
+  console.log("Hello, my name is Kennan");
+}
+
+console.log (interns[1].speak())
```

Comment:
Here the `console.log` prints `undefined` because the function isn't returning anything.